voxblox/include/voxblox/utils/planning_utils_inl.h

```diff
@@ -131,6 +131,39 @@ void clearSphereAroundPoint(const Point& center, const FloatingPoint radius,
   }
 }
 
+// Utility function to get map bounds from an arbitrary layer.
+template <typename VoxelType>
+void computeMapBoundsFromLayer(const voxblox::Layer<VoxelType>& layer,
+                               Eigen::Vector3d* lower_bound,
+                               Eigen::Vector3d* upper_bound) {
+  FloatingPoint block_size = layer.block_size();
+
+  BlockIndexList all_blocks;
+  layer.getAllAllocatedBlocks(&all_blocks);
+
+  Point lower_bound_pt;
+  Point upper_bound_pt;
+  bool first_block = true;
+
+  for (const voxblox::BlockIndex& block_index : all_blocks) {
```

Comment:
Just a suggestion, but since you anyways limit the accuracy of this method to the block level, why not just compute the bounds on the block indices and then just compute the metric bounds in the end? It's simpler and faster.

Reply:
:+1: good call, will change!